R_scripts/licordat.R

```diff
@@ -2,7 +2,7 @@
 # Stephanie Pennington | March 2018
 
 #----- Function to parse a file and return data frame -----
-packages <- c("tidyr", "lubridate", "ggplot2", "plyr", "dplyr", "tibble")
+packages <- c("tidyr", "lubridate", "plyr", "dplyr", "tibble")
```

Comment:
Can get rid of `tibble` here--it comes along for the ride with `dplyr`.

Probably get rid of `plyr` too? What do you need it for?

Reply:
Not sure, either way don't need it anymore.